newIDE/app/src/MainFrame/EditorContainers/HomePage/InAppTutorials/GuidedLessons.js

```diff
@@ -70,9 +70,10 @@ const styles = {
 
 type Props = {|
   selectInAppTutorial: (tutorialId: string) => void,
+  filter?: ?Array<string>,
```

Comment:
may I suggest a better name for this prop to know what's inside?
Something like `filterIds` or `guidedLessonIds` (without looking at the type, I thought this was a function at first)